### PR TITLE
Renamed Puppy to Bunny

### PR DESCRIPTION
**Pull request by claw4oo**
Renamed Puppy to Bunny, using google translate for language localizations.

Basically used sublime text to find and replace every instance of Puppy I could find.

Based on an earlier chat in the ZK main chat, where a suicidal bunny fits better than suicidal puppies.
I just did the renaming to try it.
It'll probably need some testing incase I missed anything.
(sprang) @Cerin#7618
@Anarchid#7320

### CODE REVIEW

File: LuaUI/Configs/lang/units.pl.json [hunk 1]
```diff
@@ -990,7 +990,7 @@
 		"helptext" : "Jugglenaut posiada imponującą wytrzymałość i podwójne promienie grawitacyjne, które mogą przyciągać lub odpychać inne jednostki. Może również skakać, co w połączeniu z jego dużą masą pozwala mu zgniatać wrogie jednostki, skacząc na nie."
 	},
 	"jumpscout" : {
-		"name" : "Puppy",
+		"name" : "królik",
```
Comment (Anarchid) [inline LuaUI/Configs/lang/units.pl.json]: Wrong case on first letter

File: LuaUI/Configs/lang/units.ru.json [hunk 2]
```diff
@@ -1014,7 +1014,7 @@
         "name": "Drone"
     },
     "jumpscout": {
-        "helptext": "Щенок (Puppy) - быстрый рейдер-камикадзе, эффективно уничтожающий слабозащищённые цели. Находясь рядом с обломками постепенно производит из них новых щенков.",
+        "helptext": "кролик (Bunny) - быстрый рейдер-камикадзе, эффективно уничтожающий слабозащищённые цели. Находясь рядом с обломками постепенно производит из них новых щенков.",
         "description": "Шагающая ракета",
         "name": "Щенок"
```
Comment (Anarchid) [inline LuaUI/Configs/lang/units.ru.json]: Desc is updated but name is still old.

**Comment by CrazyEddieTK** [comment]
Please do not rename Puppy to Bunny.

**Comment by Aquanim** [comment]
I don't think any further renames in general are a good idea.

**Comment by claw4oo** [comment]
I'll make a forum post to get some more opinions on it. Though, this pull request can remain open until it's needed or desired

File: LuaUI/Configs/lang/units.ru.json (hunk 2 above)
Comment (claw4oo) [reply LuaUI/Configs/lang/units.ru.json]: I've updated it, but I am not currently interested in setting up transifax myself right now. Thanks for pointing it out though.

**Comment by sprangg** [comment]
I agree with no renames. The OTA names are gone which is the most important part. I'm a bit incredulous Puppy -> Bunny hasn't gone through during the 8 years it's been occasionally suggested, but its window is now past. And Puppy is a decent name anyway.

**Comment by sprunk** [comment]
I'm also disappointed that this did not happen during the rename rounds but I agree with others that this ship has sailed. Keep in mind there are multiple places that cannot be modified and will still refer to the unit as Puppy (for example youtube videos or veteran players) so a rename would come with a massive confusion cost. Our current stance is that this cost would be too high and I don't expect this stance to change anytime soon.

**Pull request closed without merging**